### ISSUE TITLE
Directus is crashing after latest update

### ISSUE THREAD

**Issue by ztob** [directus/directus#11220]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

I updated directus to latest commits in production and it started to crash. The UI is trying to load (spinner is rotating) and after about a minute it will crash and will cause also the database to crash.

![image](https://user-images.githubusercontent.com/88615762/150694553-26db5549-d51e-424f-8792-8b6871a6d7da.png)

```
Error: select "c"."column_name", "c"."table_name", "c"."data_type", "c"."column_default", "c"."character_maximum_length", "c"."is_generated", "c"."is_nullable", "c"."numeric_precision", "c"."numeric_scale", "c"."table_schema", "c"."is_identity", "c"."generation_expression", pg_get_serial_sequence(quote_ident(c.table_name), c.column_name) as serial, pg_catalog.col_description(pg_class.oid, c.ordinal_position:: int) as column_comment, COALESCE(pg.indisunique, false) as is_unique, COALESCE(pg.indisprimary, false) as is_primary, "ffk"."foreign_key_schema", "ffk"."foreign_key_table", "ffk"."foreign_key_column" from information_schema.columns c 
        LEFT JOIN pg_catalog.pg_class
          ON pg_catalog.pg_class.oid = CONCAT_WS('.', quote_ident(c.table_schema), quote_ident(c.table_name)):: regclass:: oid
          AND pg_catalog.pg_class.relname = c.table_name
       
        LEFT JOIN LATERAL (
          SELECT
            pg_index.indisprimary,
            pg_index.indisunique
          FROM pg_index
          JOIN pg_attribute
            ON pg_attribute.attrelid = pg_index.indrelid
            AND pg_attribute.attnum = any(pg_index.indkey)
          WHERE pg_index.indrelid = quote_ident(c.table_name)::regclass
          AND pg_attribute.attname = c.column_name
          AND pg_index.indnatts = 1
          LIMIT 1
        ) pg ON true
       
        LEFT JOIN LATERAL (
          SELECT
            k2.table_schema AS foreign_key_schema,
            k2.table_name AS foreign_key_table,
            k2.column_name AS foreign_key_column
          FROM
            information_schema.key_column_usage k1
            JOIN information_schema.referential_constraints fk using (
              constraint_schema, constraint_name
            )
            JOIN information_schema.key_column_usage k2
              ON k2.constraint_schema = fk.unique_constraint_schema
              AND k2.constraint_name = fk.unique_constraint_name
              AND k2.ordinal_position = k1.position_in_unique_constraint
            WHERE k1.table_name = c.table_name
            AND k1.column_name = c.column_name
        ) ffk ON TRUE
       where "c"."table_schema" in ($1) and "pg_class"."relkind" != $2 order by "c"."table_name" asc, "c"."ordinal_position" asc - Connection terminated unexpectedly
    at Connection.<anonymous> (/directus/node_modules/pg/lib/client.js:132:73)
    at Object.onceWrapper (node:events:509:28)
    at Connection.emit (node:events:390:28)
    at TLSSocket.<anonymous> (/directus/node_modules/pg/lib/connection.js:107:12)
    at TLSSocket.emit (node:events:402:35)
    at endReadableNT (node:internal/streams/readable:1343:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
18:53:22 ⚠️  Connection Error: Connection ended unexpectedly
```

### To Reproduce

on dev but is working

### What version of Directus are you using?

v9.4.3

### What version of Node.js are you using?

16

### What database are you using?

PostgreSQL 13

### What browser are you using?

Firefox, Chrome

### What operating system are you using?

Windows

### How are you deploying Directus?

CI, Docker

**Comment by benhaynes** [comment]
What version of Directus did you upgrade from?

**Comment by azrikahar** [comment]
Recently CockroachDB support was added in #10113, but there were still some kinks with it particularly affecting PostgreSQL and it's being tracked/fixed over in #11209.

With that said, I believe it is not advisable to merge the latest commit especially for production, even more so since we are moving to a two-weeks version release cycle. 

It is definitely understandable that you may want to do so to get bug fixes earlier. Unfortunately this PostgreSQL issue caused by CockroachDB support is one more reason why updating to the latest commit has an inherent risk to it.

I'll close this issue first since it is not an officially released version, but do let us know if this somehow still persists once v9.5.0 gets released.

**Comment by ztob** [comment]
hi @azrikahar 

Thank you for your reply.
You are right, I did yesterday a rollback on changes which was made about 2 weeks ago, because the last update it was about in that period.
I have checked changes in recent commits and saw that things with PG and thought the same and decided to rollback, but yes it would be great to have this better tested and fixed. Not sure why this happens, tried that query and the DB responded very fast on it.
Next time I will update only officially releases.
